signing.go

```diff
@@ -47,23 +48,56 @@ func sign() {
 	)
 	var stdErr bytes.Buffer
 	sign.Stderr = &stdErr
-	var signErr = sign.Run()
-	if signErr != nil {
+	sign.Run()
+
+	// Check if the output file was created successfully.
+	// The macOS shortcuts sign command may print error messages to stderr
+	// but still successfully create the signed file (known macOS bug).
+	if !signedShortcutCreated(outputPath) {
```

Comment:
I am open to changes that would help filter out irrelevant output. However, if you are confident that the file is still being signed, it's important to note that, to my knowledge, this issue has not been resolved in macOS Sonoma but has been fixed in Tahoe.

The problem with replacing the stderr check is that the output path may already exist as a signed Shortcut file. This check does not guarantee that the signing process successfully replaced the existing file.

This would mean that if a previously signed Shortcut file exists, we wouldn't output anything from the Shortcuts binary, even if it outputs to stderr, complaining it can't sign for some reason.